### PR TITLE
feat: support hover tooltip for scss

### PR DESCRIPTION
**Pull request by balaji-sivasakthi**
fix: https://github.com/microsoft/vscode/issues/194707 #271

### CODE REVIEW

**Comment by balaji-sivasakthi** [comment]
@microsoft-github-policy-service agree [company="The Reciprocal Solutions"]

**Comment by balaji-sivasakthi** [comment]
@aeschli ?

**Review by aeschli — CHANGES_REQUESTED** [review]

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/services/cssHover.ts [hunk 1]
```diff
@@ -38,13 +38,23 @@ export class CSSHover {
...
 
 		for (let i = 0; i < nodepath.length; i++) {
 			const node = nodepath[i];
 
+			if (node instanceof nodes.Media){
+					const regex = /@media[^\{]+/g;
+					const matches = node.getText().match(regex);
+					flagOpts = {
+						isMedia:true,
+						text:matches?.[0].toString()!
```
[the remark this answers is not in the document]
Comment (balaji-sivasakthi) [reply src/services/cssHover.ts]: ok, I removed it.

File: src/services/cssHover.ts [hunk 2]
```diff
@@ -38,13 +38,23 @@ export class CSSHover {
 		 * Build up the hover by appending inner node's information
 		 */
 		let hover: Hover | null = null;
+		let flagOpts:{text:string;isMedia:boolean};
 
 		for (let i = 0; i < nodepath.length; i++) {
 			const node = nodepath[i];
 
+			if (node instanceof nodes.Media){
```
[the remark this answers is not in the document]
Comment (balaji-sivasakthi) [reply src/services/cssHover.ts]: Is this ok now, I runned default prettier.

[the remark this answers is not in the document]
Comment (balaji-sivasakthi) [reply src/services/cssHover.ts]: Should I rise an issue for format script. It might helps other engineers to follow common format configuration.

**Comment by balaji-sivasakthi** [comment]
@aeschli have any prettier config?

**Comment by aeschli** [comment]
No prettier, just the built-in TypeScript compiler with default settings.

**Comment by balaji-sivasakthi** [comment]
@aeschli any update?

**Review by aeschli — APPROVED** [review]

**Review by chrmarti — APPROVED** [review]

**Pull request merged**